packages/contracts-bedrock/contracts/test/OptimismPortal.t.sol

```diff
@@ -314,6 +313,81 @@ contract OptimismPortal_Test is Portal_Initializer {
         vm.expectRevert("L2OutputOracle: No output found for that block number.");
         assertEq(op.isBlockFinalized(checkpoint + 1), false);
     }
+
+    function test_finalizeWithdrawalTransaction_differential(
```

Comment:
Is correct to call this `_differential`? It looks to me like it's doing differential testing on the `outputRoot` and `withdrawalHash`, but not on a different implementation of `finalizeWithdrawalTransaction`.

Reply:
The proofs and output roots are determined by the fuzzer so I would say yes that its a differential test of finalize withdrawal tx